doc/ADR/release_process.md

```diff
@@ -26,19 +26,41 @@ anything more, but the process is described below for reference:
 
 3. `.github/workflows/regular-release.yaml` gets triggered on the `push`
    of the tag, and creates a Github release of the version that was
-   described in thetag pushed (so, tag `cbmc-5.15.20` is going to
+   described in the tag pushed (so, tag `cbmc-5.15.20` is going to
    create the release titled `cbmc-5.15.20` on the release page).
 
 4. `.github/workflows/release-packages.yaml` gets triggered automatically
    at the creation of the release, and its job is to build packages for
    Windows, Ubuntu 18.04 and Ubuntu 20.04 (for now, we may support more
    specific Ubuntu versions later) and attaches them (after it has finished
-   building them) to the release page as release artifacts. 
+   building them) to the release page as release artifacts.
 
    It also makes a PR to the homebrew repository that updates the version
    of CBMC in homebrew. That's then approved and merged by the maintainers
    of homebrew.
 
+## Versioning
+
+We adopt an approach approximating SemVer. That is, our version numbers should
+be major.minor.patch. Regular releases (as of 2022-06-23: every other week)
+should normally increment the minor version number. This is also where we can
+deprecate undesired features, i.e., mark as deprecated, warn existing users.
+
+Patch version increments should not be done as part of the regular releases,
+and instead should be reserved for critical bugfixes. Don't make a patch
+release if you're not confident your release contains nothing but bugfixes.
```

Comment:
We sometimes do patch releases when we fix any problems with our CI infrastructure . Does it make sense to mention that?

Reply:
Done.